buildspec.yml

```diff
@@ -0,0 +1,23 @@
+version: 0.2
```

Comment:
Add a comment describing how/where this file is used so as not to confuse repo users.

Reply:
We can also not use a buildspec.yml to avoid confusion as it's not use for launching the demo in a customer account

Reply:
No, I think it's okay. Was just thinking a comment would help eliminate any potential confusion as to the purpose of the file.